src/Sulu/Bundle/AdminBundle/Resources/js/containers/Toolbar/withToolbar.js

```diff
@@ -1,33 +1,38 @@
 // @flow
 import {autorun} from 'mobx';
-import React from 'react';
-import type {ComponentType} from 'react';
+import type {Component} from 'react';
 import {buildHocDisplayName} from '../../services/react';
 import type {ToolbarConfig} from './types';
 import toolbarStorePool, {DEFAULT_STORE_KEY} from './stores/ToolbarStorePool';
 
 export default function withToolbar(
-    Component: ComponentType<*>,
+    Component: Class<Component<*, any>>,
```

Comment:
The `any` can be replaced by `*`. Is a little bit more specific (code further down is still checked, whereby any disables all checks after it.